bom/pom.xml

```diff
@@ -62,6 +62,7 @@
     <truth.version>1.1.3</truth.version>
     <vertx.version>4.3.2</vertx.version>
     <zookeeper.image.name>confluentinc/cp-zookeeper:7.2.1</zookeeper.image.name>
+    <testcontainers.version>1.17.4</testcontainers.version>
```

Comment:
is this a required addition or have you simply playing around with it?

Reply:
It is required as we do junit tests against mongodb 6.0.
The bug fixed at testcontainers is https://github.com/testcontainers/testcontainers-java/issues/5768

Reply:
ok, can you then please move the property up so that it adheres to the alphabetical ordering?

You might also want to remove the *draft* status from this PR then ... :-)

Reply:
sure, done :)